### ISSUE TITLE
Some questions

### ISSUE THREAD

**Issue by marksev1** [edfungus/Crouton#3]
Can you maybe please show me where in the code the color scheme could be changed (for example to make a dark theme).

Also is there a possibility to add an advanced graph, with X and Y axis (visually like Grafana-style) to the dashboard or are just pie charts available.

**Comment by edfungus** [comment]
Most of the colors and styles that span the whole app is in **style.less** in **/public/common/css**

There is a line chart in the demo but I forgot to add it to the readme. I went I ahead and added it to the readme. While it can do multiple lines, it is still pretty basic. 

Let me know if you have other questions!

**Comment by marksev1** [comment]
Aha so it can display for example two mqtt temperature values (for example one from a sensor outside and one from a sensor inside, with different colour lines I presume?). Does it have any persistance, or maybe a configurable time of persistance (1day, 1 week) and an interval on at which times it should sample and draw the data?

Are you maybe also thinking about a rgb color picker (color wheel), that would be essential for my ws2812 leds :D

**Comment by edfungus** [comment]
It should be able to draw two lines from different  sources however I have not tried it. Other than initializing the graph with two lines, each device will send values with the same x axis but with one y value (data point) for its line? Sounds like Crouton should be able to handle it but not sure. 

There is no persistence right now since the whole project runs only on the front end (essentially). That was in the backlog for the project but would require the setting up of user management in order securely store the data.

A color picker has been requested before but no implementation yet.

Overall, I haven't worked on this project a lot since the new year. Not that I do not want to, but my time is fairly limited. However, if there is a lot of interest in this project, I don't mind adding features and expanding it!

:)

**Comment by marksev1** [comment]
Cool, visually your solution looks great to me so I have lots of ideas on how to improve it :D.

But yeah lets wait on a better time. Don't want to push you, if you are busy :)

**Comment by gilav** [comment]
Hello

I wanted to know if it's possible to reduce the font-size used in the crouton-simple-text value field. I display a dateTime and the text is way too big.

by the way  this is a really nice tool...
gilles

**Comment by edfungus** [comment]
@gilav, I opened a new issue for you here #33

**Comment by gilav** [comment]
Hello Edmund 

Thanks for the reply on the font size.

I have some other questions:
- how are the endPoints control ordered on the dashboard? I think it does not reflect the order from the deviceInfo json configuration file.
- would it be possible to visually group them per device? Or have some kind of separator? On the  dashboard that I created I have 2 devices, and it is impossible to determine which control is on which device.
- Is it possible (and interesting) to have a control like the crouton-simple-togle but with 3 states: Off/starting-or-stopping-state/On? I ask this because I try to control long-running process on a linux box/raspberry, using a python client + PM2 (at this time). And I would like to be able to see all the 'start-command/starting/started' and  'stop-command/stopping/stopped'  on one control. Sometime the process don't start, or don't stop well.

Thanks and best regards

Gilles
